webpack/base.js

```diff
@@ -2,24 +2,30 @@ import webpack from 'webpack'
 import path from 'path'
 
 export default {
-
-  entry: ['./src/client'],
-
+  entry: ['babel-polyfill', './src/client'],
+  resolve: {
+    modules: ['../src', '../node_modules'].map(p => path.resolve(__dirname, p)),
```

Comment:
Sounds weird to me too, but apparently webpack ignore the babel resolver, I didn't searched really much for the cause. I agree it would be better without that.